File: app/src/main/java/com/example/myspacevendor/model/ServerResponse.java
package com.example.myspacevendor.model;

import com.example.myspacevendor.data.Shop;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ServerResponse {

    @SerializedName("error")
    @Expose
    private Boolean error;

    @SerializedName("message")
    @Expose
    private String message;


    @SerializedName("shops")
    @Expose
    private List<Shop> shopList;


    @SerializedName("shop")
    @Expose
    private Shop shop;

    public Boolean getError() {
        return error;
    }

    public void setError(Boolean error) {
        this.error = error;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public List<Shop> getShopList() {
        return shopList;
    }

    public void setShopList(List<Shop> shopList) {
        this.shopList = shopList;
    }

    public Shop getShop() {
        return shop;
    }

    public void setShop(Shop shop) {
        this.shop = shop;
    }
}

File: app/src/main/java/com/example/myspacevendor/model/ServerResponse.kt
package com.example.myspacevendor.model

import com.example.myspacevendor.data.Booking
import com.example.myspacevendor.data.Shop
import com.example.myspacevendor.data.Vendor_Name_List
import com.example.myspacevendor.data.slot.SlotData
import com.example.myspacevendor.data.user.User
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ServerResponse {
    @SerializedName("error")
    @Expose
    var error: Boolean? = null

    @SerializedName("message")
    @Expose
    var message: String? = null

    @SerializedName("shops")
    @Expose
    var shopList: List<Shop>? = null

    @SerializedName("vendors")
    @Expose
    var vendor_name_lists: List<Vendor_Name_List>? = null

    @SerializedName("shop")
    @Expose
    var shop: Shop? = null


    @SerializedName("user")
    @Expose
    var user: User? = null

    @SerializedName("bookings")
    @Expose
    var bookingList: List<Booking>? = null



    @SerializedName("slots")
    @Expose
    var slotList: List<SlotData>? = null
}